app/home/[lang]/session.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import { signOut, useSession } from "next-auth/react";
+import { getUser } from "@/server/admin-function/get-user";
+import Image from "next/image";
+
+export const Session = () => {
+  const { data: session } = useSession();
+  return (
+    <>
+      {session ? (
+        <>
+          <Image
+            alt="User Image"
+            src={session.user?.image as string}
+            className="h-20 w-20 rounded-full"
+            fill
+          ></Image>
+          <h1 className="text-3xl font-bold text-green-500">
+            Welcome back, {session.user?.name}
+          </h1>
+          <h2 className="text-3xl font-bold text-blue-500">
+            UserId: {session.user?.id}
+          </h2>
+          <p className="text-2xl font-semibold">{session.user?.email}</p>
+          <button
+            onClick={() => signOut()}
+            className="rounded-lg border border-black bg-red-400 px-5 py-1"
+          >
+            Sign Out
+          </button>
+          <button
+            onClick={() => getUser("local-108", "ncastonguay01@gmail.com")}
+            className="rounded-lg border border-black bg-red-400 px-5 py-1"
+          >
+            Test
+          </button>
+          {/* <button
+            onClick={() => addUser("local-108", 'test@gmail.com', "TEST ID", "Test User")}  
+            className="rounded-lg border border-black bg-blue-400 px-5 py-1"
+          >
+            HAD UZER TWO DATA
+          </button> */}
+          <p>{session.user?.role}</p>
+        </>
+      ) : (
+        <>
+          <h1 className="text-3xl font-bold text-red-500">
+            {"You're not logged in"}
+          </h1>
+
+          <div className="flex space-x-5"></div>
+        </>
+      )}
+    </>
+  );
+};
```

Comment:
All the text, "You're not logged in", "Welcome back", etc.. is hardcoded, use uiContent to make it multi-language

Reply:
Je viens de réaliser que la page au complet est peut-être juste une page test et devrait être enlevé d'une mise en production

Reply:
est-ce que lon la laisse la pour de futur implement qui vont necessiter le login? puisque google ne marche pas sur local-108.lcoalhost:3000

Reply:
je préfèrerais une solution alternative qui nous permet de le tester dans un contexte plus normale, j'enlèverais quand même ça c'est pas une bonne pratique de faire des pages de test sur notre landing page officiel